modules/test/playwright/tests/users-admin-web/permissions/userGroupPermissions.spec.ts

```diff
@@ -0,0 +1,231 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import {rolesPagesTest} from '../../../fixtures/rolesPagesTest';
+import {usersAndOrganizationsPagesTest} from '../../../fixtures/usersAndOrganizationsPagesTest';
+import getRandomString from '../../../utils/getRandomString';
+import {
+	performLoginViaApi,
+	performLogout,
+	userData,
+} from '../../../utils/performLogin';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginTest(),
+	rolesPagesTest,
+	usersAndOrganizationsPagesTest
+);
+
+test(
+	'Can assign role to users of different sites',
+	{tag: '@LPD-55670'},
+	async ({
+		apiHelpers,
+		blogsPage,
+		page,
+		roleAssigneesPage,
+		rolePage,
+		roleUserGroupSelectorPage,
+		rolesPage,
+		usersAndOrganizationsPage,
+	}) => {
+		const site = await apiHelpers.headlessSite.createSite({
+			name: getRandomString(),
+		});
+
+		apiHelpers.data.push({id: site.id, type: 'site'});
+
```

Comment:
instead of this you can import isolatedSite (or something like this) and have the site as fixture (check accountPermissions.spec.ts -> the last test)